docs/user-manual/en/amqp-broker-connections.md

```diff
@@ -0,0 +1,238 @@
+
+To define an AMQP broker connection, add an `<amqp-connection>` element within the `<broker-connections` element in the `broker.xml` configuration file. For example:
+
+```xml
+<broker-connections>
+    <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-broker" retry-interval="100" reconnect-attempts="-1" user="john" password="doe">
+         ...
+    </amqp-connection>
+</broker-connections>
+```
+
+- `uri`: tcp://host:myport (this is a required argument)
+- `name`: Name of the connection used for management purposes
+- `user`: User name with which to connect to the endpoint (this is an optional argument)
+- `password`: Password with which to connect to the endpoint (this is an optional argument)
+- `retry-interval`: Time, in milliseconds to wait before retrying a connection after an error. The default value is `5000`.
+- `reconnect-attempts`: default is -1 meaning infinite
+- `auto-start` : Should the broker connection start automatically with the broker. Default is `true`.
+
+*Important*: The target endpoint needs permission for all operations that you configure. Therefore, If you are using a security manager, ensure that you perform the configured operations as a user with sufficient permissions.
+
+# AMQP Server Connection elements
+The following types of elements are supported on a AMQP server connection:
+
+* Senders
+    * Messages received on specific queues are transferred to another endpoint
+* Receivers
+    * The broker pulls messages from another endpoint
+* Peers
+    * The broker creates both senders and receivers on another endpoint that knows how to handle them. Currently, this is implemented by Apache Qpid Dispatch.
+* Mirrors
+    * The broker uses an AMQP connection to another broker and duplicate messages and sends acknowledgements over the wire.
+
+## Senders and Receivers
+It is possible to connect an ActiveMQ Artemis broker to another AMQP endpoint simply by creating a sender or receiver broker connection element.
+
+For a `sender`, the broker creates a message consumer on a queue that sends messages to another AMQP endpoint.
+
+For a `receiver`, the broker creates a message producer on an address that receives messages from another AMQP endpoint.
+
+Both elements work like a message bridge. However, there is no additional overhead required to process messages. Senders and receivers behave just like any other consumer or producer in ActiveMQ Artemis.
+
+You can configure senders or receivers for specific queues. You can also match senders and receivers to specific addresses or _sets_ of addresses, using wildcard expressions. When configuring a sender or receiver, you can set the following properties:
+
+- `match`: Match the sender or receiver to a specific address or __set__ of addresses, using a wildcard expression
+- `queue-name`: Configure the sender or receiver for a specific queue
```

Comment:
Are match and queue-name common to all elements?

if so, maybe point that out up front, as in all elements operate on queues via a single instance or a match.

btw: why not have match=x where x can be a wildcard. when it is a simple name then it is the same as queue-name. as in, I don't see the need for two attributes. what did I miss?

Reply:
match is not available on mirror ATM. but it's available on the others.

I had plans to implement it on mirror, but had some difficulties at the moment.